app/views/users/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Users#index</h1>
```

Comment:
Are we going to want to list users at all? If not this file and the corresponding route/controller action should be removed.

Reply:
I did this just to function as a generic landing page temporarily, but you're right that it's not needed.

Reply:
That makes sense, it should be fine to leave this in for now.